src/components/cards/standard/StandardCardDisplay.tsx

```diff
@@ -0,0 +1,176 @@
+import { CompositionMethod, useComposedCssClasses } from '../../../hooks/useComposedCssClasses';
+import {
+  ThumbsFeedback,
+  builtInCssClasses as thumbsFeedbackCssClasses,
+  FeedbackType,
+  ThumbsFeedbackCssClasses
+} from '../../ThumbsFeedback';
+import { renderHighlightedValue } from '../../utils/renderHighlightedValue';
+import { CtaData } from '../../../models/StandardCardData';
+
+import { HighlightedValue } from '@yext/answers-headless-react';
+import React, { MouseEventHandler } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * The CSS class interface used for the StandardCardDisplay.
+ *
+ * @public
+ */
+interface StandardCardCssClasses extends ThumbsFeedbackCssClasses {
+  container?: string,
+  header?: string,
+  body?: string,
+  cta1?: string,
+  cta2?: string,
+  title?: string
+}
+
+/**
+ * Default Tailwind styles for the StandardCardDisplay.
+ */
+const defaultStyling: StandardCardCssClasses = {
+  container: 'flex flex-col justify-between border rounded-lg mb-4 p-4 shadow-sm',
+  header: 'flex text-neutral-dark',
+  body: 'flex justify-end pt-2.5 text-base',
+  cta1: 'whitespace-nowrap bg-primary text-white font-medium rounded-lg py-2 px-5 shadow',
+  cta2: 'whitespace-nowrap bg-white text-primary font-medium rounded-lg py-2 px-5 mt-2 shadow',
+  title: 'text-lg font-medium',
+  feedbackButtonsContainer: 'flex justify-end mt-4 text-sm text-gray-400 font-medium',
+  thumbsUpIcon: thumbsFeedbackCssClasses.thumbsUpIcon,
+  thumbsDownIcon: thumbsFeedbackCssClasses.thumbsDownIcon
+};
```

Comment:
Is the removal of many of the classes from the interfaces intentional? We have generally followed the convention that all styling of the component should be customizable from the CSS class interface. For example, titleLink and ctaContainer are no longer customizable

Reply:
I forgot to mention that in the commit message, but yes, this was intentional. Davish and I went through and took out a number of styling config options. The purpose of this Component is to show someone how to write a Card. We should show them how to support CSS customization. But, we don't need to make every part of the DOM customizable to do that.